vktrace/vktrace_replay/vkreplay_objmapper_class_defs.h

```diff
@@ -67,7 +67,7 @@ class gpuMemory {
         PageGuardChangedBlockInfo *pChangedInfoArray = (PageGuardChangedBlockInfo *)pSrcData;
         if (pChangedInfoArray[0].length) {
             PBYTE pChangedData = (PBYTE)(pSrcData) + sizeof(PageGuardChangedBlockInfo) * (pChangedInfoArray[0].offset + 1);
-            DWORD CurrentOffset = 0;
+            size_t CurrentOffset = 0;
             for (DWORD i = 0; i < pChangedInfoArray[0].offset; i++) {
```

Comment:
I don't understand what this code is doing, but using a DWORD loop index with a size_t upper bound seems wrong from a type safety point of view.  But I have a hard time imagining this loop iterating enough to require a size_t, so I wonder if `pChangedInfoArray[0].offset` is really the right value to use for the loop upper bound.  As I said, I can't tell what this code is supposed to do, but I think it would be good to take a closer look at it.  It may also be worth adding a comment or two to the function to explain it.